.github/workflows/docs-guide-test.yml

```diff
@@ -1,10 +1,5 @@
 name: Test Docs Guide
-on:
-  push:
-    branches:
-      - main
-      - '1.*'
-  pull_request:
+on: [pull_request]
```

Comment:
Why not just `on: pull_request`?

Reply:
Ah didn't realize it could be that. Seems fine either way though